src/components/link/link.pw.tsx

```diff
@@ -0,0 +1,420 @@
+import { expect, test } from "@playwright/experimental-ct-react17";
+import React from "react";
+import { HooksConfig } from "../../../playwright";
+import {
+  getDataElementByValue,
+  icon,
+  link,
+  tooltipPreview,
+} from "../../../playwright/components";
+import { skipLink, linkChildren } from "../../../playwright/components/link";
+import { ICON } from "../../../playwright/components/locators";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import {
+  checkAccessibility,
+  containsClass,
+} from "../../../playwright/support/helper";
+import Box from "../../../src/components/box";
+import Link, { LinkProps } from "../../../src/components/link";
+import {
+  LinkComponent,
+  LinkComponentAsButton,
+} from "../../../src/components/link/components.test-pw";
+/* eslint-disable jest/valid-expect, jest/no-disabled-tests, no-unused-expressions */
+
+const testData = [CHARACTERS.DIACRITICS, CHARACTERS.SPECIALCHARACTERS];
+const testPlaywright = "test-playwright";
+
+test.describe("check props for Link component", () => {
+  testData.forEach((className) => {
+    test(`should render className using ${className} special characters`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<LinkComponent className={className} />);
+      const linkWrapperElement = await link(page);
+      await containsClass(linkWrapperElement, className);
+    });
+  });
+
+  test("should render disabled when rendering a button", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<LinkComponentAsButton disabled />);
+    const buttonElement = await linkChildren(page);
```

Comment:
**(question)**: do we need to have await before linkChildren(page)? Or can we just have` const buttonElement = linkChildren(page);`

Reply:
Thanks, this is already covered by Dan's comment above 👍